### PR TITLE
Sre 2848 tompos shellcheck

### PR DESCRIPTION
**Pull request by tomposmiko**

### CODE REVIEW

File: orca/scripts/install_orca.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ function main {
 
 	mkdir -p /opt/liferay/orca
 
-	cd /opt/liferay/orca
+	cd /opt/liferay/orca || exit 1
```
Comment (zsoltbalogh) [inline orca/scripts/install_orca.sh:14]: Can you please move all of these to lcd as I showed this morning?

File: orca/scripts/orca.sh [hunk 2]
```diff
@@ -108,38 +108,43 @@ function command_up {
 
 	if [ -d /opt/liferay/passwords ]
 	then
-		for service in $(ls /opt/liferay/passwords)
+		for service in /opt/liferay/passwords/*
 		do
+			[[ -e ${service} ]] || break
```
Comment (zsoltbalogh) [inline orca/scripts/orca.sh:113]: We don't use this pattern, I guess its needed to check if there are files in the directory. Let's use find in the for loop instead.

**Review by zsoltbalogh — CHANGES_REQUESTED** [review]
I had 2 minor change requests, can oyu please check them?

File: orca/scripts/install_orca.sh (hunk 1 above)
Comment (tomposmiko) [reply orca/scripts/install_orca.sh]: Done.

File: orca/scripts/orca.sh (hunk 2 above)
Comment (tomposmiko) [reply orca/scripts/orca.sh]: I changed it to a while loop based solution.

**Comment by zsoltbalogh** [comment]
Forwarded to https://github.com/brianchandotcom/liferay-docker/pull/380

**Pull request closed without merging**